ofctrl/ofSwitch.go

```diff
@@ -35,6 +36,7 @@ type OFSwitch struct {
 	dropAction   *Output
 	sendToCtrler *Output
 	normalLookup *Output
+	portMux      sync.Mutex
 	outputPorts  map[uint32]*Output
```

Comment:
Any reason to not use concurrent map? 
https://github.com/contiv/ofnet/blob/master/ofctrl/ofSwitch.go#L41

Reply:
Good idea!

Go actually has a build in concurrent map now:  https://golang.org/pkg/sync/#Map

(but we can't use it until we're compiling with 1.9)

Reply:
@rchirakk: This is a good idea. Can you modify the code to use the concurrent map, please?

Reply:
didn't use concurrent map because 
1.  to keep the change to minimum.
2. concurrent map requires string as key
3. concurrent map is implemented as 32 maps with mutex, overkill for this case.

Reply:
@rchirakk: That makes sense. Thank you.